TodoApp/Tests/TodoApp.Dependency.Tests/Configs/UnityConfigTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using NUnit.Framework;
+using TodoApp.Contracts.Bootstraps;
+using TodoApp.Contracts.Containers;
+using TodoApp.Dependency.Tests.Containers;
+
+namespace TodoApp.Dependency.Tests.Configs
+{
+    [TestFixture]
+    public class UnityConfigTests
+    {
+        private static readonly Type[] ExcludedTypes = 
+        {
+            typeof(IBootstrap),
+            typeof(IDependencyContainer)
+        };
+
+        [Test]
+        public void RegisterDependencies_RegistersRequiredDependencies()
+        {
+            var container = new TestContainer();
+            var contractTypes = typeof(IBootstrap).Assembly.GetTypes();
+            var types = contractTypes
+                .Where(IsNotExcludedInterface)
+                .Append(typeof(HttpRequestMessage));
+
+            DependencyConfig.RegisterDependencies(container);
+
+            Assert.That(container.GetRegisteredTypes(), Is.EquivalentTo(types));
+        }
+
+        private static bool IsNotExcludedInterface(Type type)
```

Comment:
Isn't more of a `IsNotExcludedAndInterface` or something? Just a nitpicking detail and something to consider.